lib/sles4sap/ipaddr2.pm

```diff
@@ -1915,4 +1946,244 @@ sub ipaddr2_test_other_vm {
     }
 }
 
+=head2 ipaddr2_clean_network_peering
+
+    ipaddr2_clean_network_peering();
+
+Cleanup the network peering if needed.
+
+=over
+
+=back
+
+=cut
+
+sub ipaddr2_clean_network_peering {
+    return unless get_var("IBSM_RG");
+    my $source_group = ipaddr2_azure_resource_group();
+    my $ibs_mirror_resource_group = get_required_var('IBSM_RG');
+
```

Comment:
This lib function seems mainly to be here to call another lib function `ipaddr2_azure_resource_group` (this is fine) and read a variable (not ok, I would like to try to avoid this library to directly read variables, particularly when variables are not "owned" by this library itself), to be able to call an external lib function (not wrong but it introduce a dependency, that again is not wrong but could potentially lead to maintenance work in the future).